alternative_backend/common/auth.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import permissions
+
+
+ACCESS_KEY = "123dupachujpietaszekalternativelongboardspiotrekkurwa"
```

Comment:
Nie commituj kluczy i nie dodawaj ich w repozytorium. Użyj zmiennych środowiskowych, lub argumentów przekazanych do runserver. 
A przede wszystkim nie commituj przekleństw bo to nie profesjonalne.